tests/test_transformations.py

```diff
@@ -115,3 +117,17 @@ def test_difference():
         transformations_over_time[0].iloc[0].inverse_transform(pred).squeeze(),
         atol=1e-3,
     ).all()
+
+
+def test_holiday_transformation() -> None:
+    X, y = generate_sine_wave_data()
+    new_index = pd.date_range(start="1/1/2018", periods=len(X))
+    X.index = new_index
+    y.index = new_index
+
+    splitter = ExpandingWindowSplitter(initial_train_window=400, step=400)
+    transformations = [AddHolidayFeatures(["US", "DE"])]
+
+    transformations_over_time = train(transformations, X, y, splitter)
+    pred = backtest(transformations_over_time, X, y, splitter)
+    assert (np.isclose((X.squeeze()[pred.index]), (pred["sine"]))).all()
```

Comment:
could you re-work this test to demonstrate that the output is valid? for example, check if holiday == True for Christmas?